### ISSUE TITLE
mvn repository version

### ISSUE THREAD

**Issue by Kevgilmore** [SafeChargeInternational/safecharge-java#25]
The version from the readme is not available on maven respository.

see https://mvnrepository.com/artifact/com.safecharge/safecharge-sdk-java

thanks

**Comment by Scoty** [comment]
The current version is a SNAPSHOT version. There is an internal discussion about which features will make it into the GA version. As soon the discussion is finished we will release the GA 1.1.0 version.

**Comment by DopplerEffect666** [comment]
Now we released stable version 1.1.0 containing changes till dotay on maven central and also here in GitHub. Now you can add the following in the POM file:

```
<!-- https://mvnrepository.com/artifact/com.safecharge/safecharge-sdk-java -->
<dependency>
    <groupId>com.safecharge</groupId>
    <artifactId>safecharge-sdk-java</artifactId>
    <version>1.1.0</version>
</dependency>
```
